docs/install/README.md

```diff
@@ -392,29 +392,43 @@ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3
 ### Install K8ssandra Operator
 The GitHub Actions for the project are configured to build and push a new operator image to Docker Hub whenever commits are pushed to `main`. 
 
-See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) on Docker Hub for a list of availabe images.
+See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) on Docker Hub for a list of available images.
 
-Create a kustomization directory that builds from the `main` branch:
+If you wish to install the default installation, one can install with kubectl:
+
+```console
```

Comment:
I think we should add a warning/note that this will install the `latest` tag which is discouraged.

Reply:
You mean return the "main" notion?

Reply:
I am saying that we ought to point out that this will deploy `k8ssandra/k8ssandra-operator:latest` and using `latest` is discouraged. There is already an example further down that sets the image tag. We could just reference that for specifying the image version.